modules/system/holoportos/udiskie.nix

```
@@ -0,0 +1,40 @@
+{ config, pkgs, ... }:
+let
+  wrapSystemdScript = (
+    program: ''
+      source ${config.system.build.setEnvironment}
+      exec ${program}
+    ''
+  );
+in rec {
+  environment.systemPackages = with pkgs; [
+    udiskie
+    udisks2
+  ];
+  services = {
+    udisks2.enable  = true;
+  };
+  systemd.user.services = {
+    udiskie = {
+      description = "Mounts disks in userspace with udisks";
+      serviceConfig = {
+        Type    = "simple";
```

```suggestion
        Type = "simple";
```

Modern upstream Nixpkgs identation doesn't use aligning.

---

Also, `simple` is the default, so this doesn't need to be specified. See: https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=